### ISSUE TITLE
Add alt-text on the deployment-logo

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4102]
## Description:
The deployment-image is set when navigating to Settings-> General. Once uploaded, the image is visible in the map-view, the activity-view and the settings-view. This image has no alt-text so we need to add that.

## Todo:
Search for the deployment-logo <img> tag in the code. Hint! The class used is "deployment-logo" so you can easily find it! Add an alt-text saying "Deployment-logo" to each element.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-854

**Comment by Somaru-chan** [comment]
Hi @Angamanga . Can I work on this?

**Comment by Sunairaa** [comment]
Hi @Angamanga can i work on this task? i am an outreachy candidate.

**Comment by hyounes4560** [comment]
Hi @Angamanga can I work on this task?

**Comment by ulomau** [comment]
Pls can i claim this issue

**Comment by RaissaMariaB** [comment]
Hello, can i work on this task? i am an outreachy applicant

**Comment by treasurechristain** [comment]
@Angamanga I'm Amadikwa Joy N. An outreachy Applicant. Can I work on this issue please.

**Comment by Anisha-28** [comment]
@Angamanga Hi I'm Anisha i can work on this issue

**Comment by bimbolabuari** [comment]
@rowasc @Angamanga, 
Hi, I'm Abimbola an outreachy applicant and I would love to work on this issue

**Comment by LavellTheDev** [comment]
Is this issue still open to 1st timers? If so, I would like to work on this.

**Comment by Angamanga** [comment]
@Somaru-chan Please go ahead with this issue if you are still interested 🙏 .

**Comment by Somaru-chan** [comment]
> @Somaru-chan Please go ahead with this issue if you are still interested 🙏 .

Thanks @Angamanga ! Will open a PR soon.

**Comment by Somaru-chan** [comment]
Hi @Angamanga . Would you mind taking a look to see if anything needs modification? Thanks!
PR: https://github.com/ushahidi/platform-client/pull/1586

**Comment by Angamanga** [comment]
@Somaru-chan Thank you! I will review shortly!
